### DISCUSSION TITLE
beats so-allow how to view IPs

### DISCUSSION THREAD

**Discussion by sinnetll19** [Security-Onion-Solutions/securityonion#3385] [Unsupported Versions] [answered]
when running so-allow and choosing option b to open the ports for winlogbeat. How do you view the list of IPs previously allowed?. Running so-allow-view only shows the analyst allowed IPs.

**Comment by presianbg** [comment, marked as answer]
Hi,

You might try with:

`sudo so-firewall includedhosts beats_endpoint`

The other roles are defined here:
https://github.com/Security-Onion-Solutions/securityonion/blob/2df871adcd0cd7e56077dc548d7c0ab048c5baa3/salt/common/tools/sbin/so-allow

Or you can see them with:
```
so-firewall listhostgroups
```

Cheers,
PY

**Reply by sinnetll19** [reply]
this works great. thanks
